thrift/src/main/thrift/atoms/storyquestions.thrift

```diff
@@ -32,4 +32,5 @@ struct StoryQuestionsAtom {
   4: optional list<QuestionSet> editorialQuestions
   5: optional list<QuestionSet> userQuestions
   6: optional shared.NotificationProviders notifications
+  7: optional bool closed = false
```

Comment:
I asked Chris M if he wanted to set a datetime rather than a flag, he opted for this.
Having this flag at the top level instead seems reasonable to me